1-exercises/A-this/exercise-2.js

```diff
@@ -11,7 +11,9 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function (){
```

Comment:
You need to modify changeAddress function in the way that it is going to take a new address as a parameter and update the address stored inside the object with a new value.